backend/config.py

```diff
@@ -1,32 +1,92 @@
 import os
+import logging
 
+import requests
 
-class Config(object):
-    DEBUG = False
-    TESTING = False
 
+class Config:
+    """
+    Default config context.
+    """
 
-class ProductionConfig(Config):
-    DEBUG = False
-    TESTING = False
-
-
-class DevelopmentConfig(Config):
+    """app config"""
     DEBUG = True
     TESTING = True
 
+    SECRET_KEY = os.environ["SECRET_KEY"]
+    BASE_URL = os.environ["BASE_URL"]
+
     """db config"""
-    DB_USER = os.environ.get("DB_USER")
-    DB_PASSWORD = os.environ.get("DB_PASSWORD")
-    DB_HOST = os.environ.get("DB_HOST")
-    DB_PORT = os.environ.get("DB_PORT") or 5432
-    DB_NAME = os.environ.get("DB_NAME")
+    DB_HOST = os.environ["DB_HOST"]
+    DB_PORT = os.environ["DB_PORT"] or 5432
```

Comment:
this `or 5432` also should be removed but aws lambda does not have `DB_PORT` env variable so I have to fix that first in terraform and then we can remove it :)